tools/llm_bench/llm_bench_utils/parse_json_data.py

```diff
@@ -2,78 +2,67 @@
 # Copyright (C) 2023-2025 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+def create_base_prompt(json_data, key='prompt'):
+    prompt_data = {}
+    if key in json_data:
+        if json_data[key] != "":
+            prompt_data[key] = json_data[key]
+        else:
+            raise RuntimeError(f"== {key} should not be empty string ==")
+    else:
+        raise RuntimeError(f"== key word '{key}' does not exist ==")
+    return prompt_data
+
 
 def parse_text_json_data(json_data_list):
     text_param_list = []
     for json_data in json_data_list:
-        if 'prompt' in json_data:
-            if json_data['prompt'] != '':
-                text_param_list.append(json_data['prompt'])
-            else:
-                raise RuntimeError('== prompt should not be empty string ==')
-        else:
-            raise RuntimeError('== key word "prompt" does not exist ==')
+        prompt_data = create_base_prompt(json_data)
+        text_param_list.append(prompt_data["prompt"])
     return text_param_list
 
 
 def parse_vlm_json_data(json_data_list):
     text_param_list = []
     for json_data in json_data_list:
-        prompt_data = {}
-        if 'prompt' in json_data:
-            if json_data['prompt'] != '':
-                prompt_data["prompt"] = json_data['prompt']
-            else:
-                raise RuntimeError('== prompt should not be empty string ==')
-        else:
-            raise RuntimeError('== key word "prompt" does not exist ==')
+        prompt_data = create_base_prompt(json_data)
+        if ("media" in json_data) and ("video" in json_data):
+            raise ValueError("only one key is available from media & video")
         if "media" in json_data:
             prompt_data["media"] = json_data["media"]
+        if "video" in json_data:
```

Comment:
Could you please align names of input args for video from json and from cmd ? `video` or `videos` in both places